front/app/containers/MainHeader/index.tsx

```diff
@@ -48,7 +53,7 @@ const Container = styled.header<{ position: 'fixed' | 'absolute' }>`
   }
 `;
 
-const ContainerInner = styled.div`
+export const ContainerInner = styled.div`
   flex-grow: 1;
```

Comment:
Exported these so I could use them in the ScrollHeader as well (I didn't need to duplicate them like the Container since these elements are the same).